test/unit/radio-helpers.spec.js

```diff
@@ -0,0 +1,84 @@
+describe('Marionette radio helpers', function() {
+
+  describe('when creating a Marionette.Object', function() {
+
+    beforeEach(function() {
+      this.clickStub1 = this.sinon.stub();
+      this.clickStub2 = this.sinon.stub();
+      this.clickStub3 = this.sinon.stub();
+      this.Object = Marionette.Object.extend({
+        radioEvents: {
+          'foo bar' : this.clickStub1
```

Comment:
@jasonLaster So after looking at this again, I don't see the problem here.  I pass 2 functions directly and one by reference, to make sure that we handle both of those cases.  Yes object2 shares the stub on the prototype, but I think the testing is still pretty unambiguous.  I just wrote it out the other way, and it makes the code a lot more verbose and (in my mind) less clear.  So I'm going to leave it unless you strongly object.